content/en/docs/reference/command-line-tools-reference/kube-scheduler.md

```diff
@@ -27,7 +27,7 @@ each Pod in the scheduling queue according to constraints and available
 resources. The scheduler then ranks each valid Node and binds the Pod to a
 suitable Node. Multiple different schedulers may be used within a cluster;
 kube-scheduler is the reference implementation.
-See [scheduling](/docs/concepts/scheduling-eviction/)
+See [scheduling](https://kubernetes.io/docs/concepts/scheduling-eviction/)
```

Comment:
Would it be possible for us to remove this hardcoded site domain name?

```suggestion
See [scheduling](/docs/concepts/scheduling-eviction/)
```

Reply:
Won't fix. This is generated out of upstream source code.

Reply:
We _could_ fix the generator code, but it'd be a very low priority thing.

Reply:
I don't think make the tool looking smart is a smart thing to do.